index.html

```diff
@@ -0,0 +1,170 @@
+
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Relaycorp - Tackling Neglected Communication Problems</title>
+        <!-- Meta tags for SEO -->
+        <meta name="description" content="Relaycorp is incubating technologies to solve humanity's most pressing and neglected communication challenges, starting with Internet blackouts.">
+        <meta name="keywords" content="Relaycorp, Awala, Letro, VeraId, Despacito, communication, Internet blackouts">
+        <!-- Favicon -->
+        <link rel="icon" type="image/png" href="favicon/favicon-48x48.png" sizes="48x48" />
+        <link rel="icon" type="image/svg+xml" href="favicon/favicon.svg" />
+        <link rel="shortcut icon" href="favicon/favicon.ico" />
+        <link rel="apple-touch-icon" sizes="180x180" href="favicon/apple-touch-icon.png" />
+        <link rel="manifest" href="favicon/site.webmanifest" />
+        <!-- Tailwind CSS via CDN -->
+        <script src="https://cdn.tailwindcss.com?v=3.3.2"></script>
+        <!-- Google Fonts -->
+        <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;600;700&family=Poppins:wght@400;600;700&display=swap" rel="stylesheet">
+        <style>
+            body {
+                font-family: 'Inter', sans-serif;
+                color: #fff;
+                h1, h2, h3 {
+                    font-family: 'Poppins', sans-serif;                    
+                }
+            }
+        </style>
+    </head>
+    <body class="bg-[#000003]">
+        <!-- Header section -->
+        <header class="w-full pt-3 lg:pt-8">
+            <div class="container mx-auto h-24 flex items-center justify-center">
+                <img src="images/relaycorp-logo.png" alt="Relaycorp Logo" class="h-8 lg:h-16">
+            </div>
+        </header>
+
+        <!-- Hero section -->
+        <section class="pt-4 md:pt-14">
+            <h1 class="text-3xl md:text-6xl lg:text-8xl px-8 font-bold text-center leading-normal lg:leading-normal">Tackling the most neglected 
+                <span class="bg-clip-text text-transparent bg-gradient-to-r from-blue-400 to-blue-700">communication problems</span></h1>
```

Comment:
I'd say "Tackling the most pressing and neglected communication problems". Both "pressing" and "neglected" are important. Maybe we'll have to reduce the font size to stick to two lines.